### DISCUSSION TITLE
Send copy of flows to Graylog (SIEM)

### DISCUSSION THREAD

**Discussion by packetThrower** [akvorado/akvorado#1387] [Q&A] [answered]
I would like to send our flows to Graylog which will eventually get sent to our SIEM. I was planning on using the Kafka input in Graylog but no messages are flowing in. So, I tried connecting with kcat just to see of I could connect to Kafka and I'm getting some errors which I think are due to MacOS and I still need to try from another OS.

Ultimately, the data will be stored in a data warehouse with Rapid7. I'm just trying to find a means to get the data there. Graylog can read Kafka. I'm sure it can work, but I'm a bit lost at the moment.

I do have the ports exposed in my docker compose.

**Comment by vincentbernat** [comment, marked as answer]
The flows are encoded into a custom protobuf message. Akvorado is not the tool you want for that. Maybe GoFlow2 can write to Kafka in JSON.

**Reply by packetThrower** [reply]
Gotcha! Thanks!

I did manage to get kcat to read messages and noticed funny chars which would be explained by the custom protobuf.
